app/site/controllers.py

```diff
@@ -42,7 +43,11 @@ def index():
                     return render_template("dashboard.html", user=user,
                                            title='Dashboard'), 200
                 return redirect(request.headers['Host'] + '/logout')
-        return render_template("index.html", title='Home'), 200
+        return render_template("index.html", title='Home',
+                               s3_cdn=get_s3_cdn_prefix(),
+                               auth0_client_id=app.config['AUTH0_CLIENT_ID'],
+                               auth0_domain=app.config['AUTH0_DOMAIN']
+                               ), 200
```

Comment:
Just a minor thing: probably that does not relate to Bootstrap, so would be helpful if there is something about that in the description.